ui/lib/replication/addon/components/replication-summary.js

```diff
@@ -66,24 +72,13 @@ export default Component.extend(ReplicationActions, DEFAULTS, {
     this.setProperties(DEFAULTS);
   },
 
-  transitionTo: computed('mode', 'replicationMode', function() {
```

Comment:
I had originally created this to handle DR secondary transition issues, but this is being reverted to how it is in master, and the transition conditions are handled in the controller/replication-mode.js